mountpoint-s3-crt-sys/UPDATING_CRT.md

```diff
@@ -0,0 +1,39 @@
+# CRT Submodules
+
+Mountpoint for Amazon S3 imports the required [AWS Common Runtime (CRT)](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html) libraries through git submodules, located under the `mountpoint-s3-crt-sys/crt` directory.
+
+Currently the imported libraries are:
+
+* `aws-c-auth`
+* `aws-c-cal`
+* `aws-c-common`
+* `aws-c-compression`
+* `aws-c-http`
+* `aws-c-io`
+* `aws-c-s3`
+* `aws-c-sdkutils`
+* `aws-checksums`
+* `aws-lc`
+* `s2n-tls`
+
+## Update Submodules
+
+The CRT submodules can be updated by following these steps:
+
+1. Update every submodule to the latest tagged release. E.g. by running:
+
+   ```sh
+   git submodule foreach -q 'git checkout `git tag -l --sort=-v:refname | head -1`'
+   ```
+
+2. Review the commit history, in particular for `aws-c-s3`, looking for changes that may affect `mountpoint-s3` (bug fixes, API changes, etc.).
```

Comment:
Should we recommend reviewing the output of `git diff --submodule [path to submodule]`?

For example:

```
❯ git diff --submodule mountpoint-s3-crt-sys/crt/aws-c-s3 

Submodule mountpoint-s3-crt-sys/crt/aws-c-s3 d7bfe60..ae6c6cc:
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  > Fix one byte range handling (#272)
  > Fix Benchmark stack (#270)
  > Update package.json (#260)
  > New `send_filepath` option, for faster uploads. (#266)
  > Skip mock server setup when tests are disabled (#267)
...
```